roles/acm_hypershift/tasks/download-cli.yml

```diff
@@ -50,4 +56,8 @@
 - name: Set hcp cli path
   ansible.builtin.set_fact:
     ah_hcp_cli_path: "{{ ah_tmp_dir }}/hcp"
+
+- name: Set oc cli path
```

Comment:
This variable is just defined here in `download-cli`, but this playbook is only called if `ah_download_cli` is true: https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/main/roles/acm_hypershift/tasks/main.yml#L12-L15. Its default value is true, but what happens if it is false? Then the usage of `ah_oc_cli_path` will fail outside this playbook because it's an undefined variable. Should we protect the code for that case?

Reply:
The assertion during the validation playbook will fail if that is undefined or false. It is covered. The user has the option to pass a specific binary already present on the system. 

https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/74bb35eb9f185cbb68f341b9e260de58b7b6da4f/roles/acm_hypershift/tasks/validations.yml#L10

Reply:
My bad, that variable was not asserted. Should be good now. Good catch!